src/frontend/komponenter/Fagsak/Simulering/SimuleringTabell.tsx

```diff
@@ -88,27 +122,46 @@ const SimuleringTabell: React.FunctionComponent<ISimuleringProps> = ({
         <>
             <SimuleringTabellOverskrift>
                 <Element>
-                    Simuleringsresultat for perioden {familieDayjs(fom).format('DD.MM.YYYY')} -{' '}
-                    {familieDayjs(tomDatoNestePeriode).format('DD.MM.YYYY')}
+                    Simuleringsresultat{' '}
+                    {perioder.length > 1 &&
+                        `for perioden ${familieDayjs(fom).format('DD.MM.YYYY')} - ${familieDayjs(
+                            tomDatoNestePeriode
+                        ).format('DD.MM.YYYY')}`}
                 </Element>
             </SimuleringTabellOverskrift>
+            {årISimuleringen.length > 1 && (
+                <div>
```

Comment:
Er dette noe lignende frem og tilbake i behandlingsresultat tidslinjen? Foreslår å gjenbruke den for likhet 🤷

Reply:
Må nok vente til Gunn kommer med et design her uansett, men fikk det inn

Reply:
Ok 👍